lib/index.js

```diff
@@ -158,6 +158,9 @@ const handleResponses = (response) => {
 }
 
 const getAllMeals = (meals) => {
+  appendToTotalCaloriesConsumed(meals);  
+  appendToTotalGoalCalories(meals); 
+  appendToTotalCaloriesRemaining(meals); 
   return meals.forEach((meal) => {
```

Comment:
I wonder if we should pull this out to and have it be a method that just takes meals?>